src/m_fixed.h

```diff
@@ -24,6 +24,14 @@
 #include <stdlib.h> // abs()
 #include <stdint.h> // int64_t
 
+// VS 2019 and later provide a [64]/[32] intrinsic
+#if defined(_MSC_VER) && _MSC_VER >= 1920 && (defined(_M_X64) || defined(_M_IX86))
+  #include <immintrin.h>
+  #define DIV64(a,b) (((a)>>FRACBITS)==INT_MIN && (b)==-1 ? 0 : _div64((a),(b),NULL))
```

Comment:
I've changed my mind after some reading on the subject, let's remove the check for `INT_MIN / -1`. Obviously there will be an integer overflow, so it's a bug anyway.

So, how do we proceed? We can merge this and I will add the GCC/Clang variant later.

Reply:
Isn't the `INT_MIN/-1` case already covered by the `(abs(a)>>14) >= abs(b)` check?

Reply:
> Isn't the `INT_MIN/-1` case already covered by the `(abs(a)>>14) >= abs(b)` check?

It doesn't seem to, the old `FixedDiv` returns `0`, not `INT_MIN` or `INT_MAX`. To be honest, I don't understand what `(abs(a)>>14) >= abs(b)` checks.